libs/shared/ui/src/lib/components/modals/modal-multi-confirmation/modal-multi-confirmation.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import { act, fireEvent, renderWithProviders, screen } from '@qovery/shared/util-tests'
+import ModalMultiConfirmation, { type ModalMultiConfirmationProps } from './modal-multi-confirmation'
+
+describe('ModalMultiConfirmation', () => {
+  let props: ModalMultiConfirmationProps
+
+  beforeEach(() => {
+    props = {
+      title: 'my title',
+      description: 'my description',
+      checks: ['one', 'two'],
+      callback: jest.fn(),
+    }
+  })
+
+  it('should render successfully', () => {
+    const { baseElement } = renderWithProviders(<ModalMultiConfirmation {...props} />)
+    expect(baseElement).toBeTruthy()
+  })
+
+  it('should render the validation checkboxes', () => {
+    renderWithProviders(<ModalMultiConfirmation {...props} />)
+
+    expect(screen.getByText('one')).toBeInTheDocument()
+    expect(screen.getByText('two')).toBeInTheDocument()
+  })
+
+  it('should not submit until checkboxes are checked', async () => {
+    renderWithProviders(<ModalMultiConfirmation {...props} />)
```

Comment:
We prefer tu use `userEvent` instead of `fireEvent` 

```suggestion
    const {  userEvent } =  renderWithProviders(<ModalMultiConfirmation {...props} />)
```